lib/RoadizCoreBundle/src/Console/UsersPasswordCommand.php

```diff
@@ -54,9 +62,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $confirmation
             )
         ) {
-            $user->setPlainPassword($this->passwordGenerator->generatePassword(
-                (int) $input->getOption('length')
-            ));
+            if ($plainPassword = $input->getOption('plain-password')) {
+                if (\mb_strlen($plainPassword) < 5) {
+                    throw new \InvalidArgumentException('Password is too short.');
```

Comment:
```suggestion
                    throw new \InvalidArgumentException('Password should be at least 12 chars long.');
```

Help user here to know why its password is too short.